### PR TITLE
report issue on toyota's website

### PR DESCRIPTION
**Pull request by dutraneto**
I noticed your work very interesting and I would like to know if I can contribute.

### CODE REVIEW

**Comment by AgtLucas** [comment]
Hi @dutraneto,

Thank you so much for reporting this!

Could you please just elaborate better which strange behaviour did you notice on Firefox?

Thank you! :)

**Comment by dutraneto** [comment]
@AgtLucas I think I made a mistake updating many gem dependencies.
Apologies for that. I'm a newbie looking for a new career.
I enjoy to cooperate with you guys because I have a long term career path

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ media: my-image-for-the-issue (make sure to place it on `/assets/img` and use th
 Here you can add any Markdown-valid content describing the accessibility issue.
 ```
 
-> :warning: Your tag must be one of the following: `"Non-perceivable"`, `"Non-operable"`, `"Non-understandable"`. To know which one to use, refer to the [POUR principles](https://www.w3.org/TR/UNDERSTANDING-WCAG20/intro.html#introduction-fourprincs-head).
+> :warning: Your tag must be one of the following: `"Non-perceivable"`, `"Non-operable"`, `"Non-understandable"`. To know which one to use, refer to the [FOUR principles](https://www.w3.org/TR/UNDERSTANDING-WCAG20/intro.html#introduction-fourprincs-head).
```
Comment (AgtLucas) [inline .github/CONTRIBUTING.md]: This is supposed to be `POUR` not `FOUR`. :)

It means **P**erceivable, **O**perable, **U**nderstandable and **R**obust.

**Comment by AgtLucas** [comment]
@dutraneto That's ok! :)

I just left another comment.

Thank you once again!

**Pull request closed without merging**